js/app.js

```diff
@@ -16,4 +18,108 @@ navLinks.forEach((link) => {
 
     navMenu.classList.remove('active');
   });
-});
\ No newline at end of file
+});
+
+const projects = [
+  {
+    id: 1,
+    name: 'Tonic',
+    description:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard  ',
+    image: '../images/work-image1.png',
```

Comment:
- [ ] [OPTIONAL] your images look broken here on my side and I think it's because you are using the incorrect path. 
![image](https://user-images.githubusercontent.com/90222110/180295180-33ee304c-a295-4e39-aabd-eeb11c505318.png)
💡 **HINT:**  can you please try to use `"./images/{your image name}"`  instead of "../images/{your image name}"

Reply:
@BertrandConxy Please this was not the case with the previous reviewers.